core/src/com/group66/game/cannon/BallsStatic.java

```diff
@@ -255,7 +255,6 @@ public void moveRowDown() {
      */
     public boolean hitsBottom() {
         for (Ball b : ballStaticList) {
-            // TODO fix the != check
```

Comment:
Yea, I was thinking about that too, I think the check can go now, but I'm not shure.
If we remove the check (which was for the cannon ball up next) we need to test real good that it still works